mms/tests/unit_tests/test_export.py

```diff
@@ -106,11 +106,15 @@ def test_export_module(tmpdir, module_dir):
     assert [f for f in zip_contents if f.endswith('.py')], 'missing service file'
 
 
-def test_export_onnx(tmpdir, module_dir, onnx_mxnet):
-    os.remove(os.path.join(module_dir, 'test-symbol.json'))
-    os.remove(os.path.join(module_dir, 'test-0000.params'))
-    empty_file(os.path.join(module_dir, 'test.onnx'))
+def test_export_module_hyphenated_basename(tmpdir, module_dir):
+    os.rename(os.path.join(module_dir, 'test-symbol.json'), os.path.join(module_dir, 'test-hyphens-symbol.json'))
+    os.rename(os.path.join(module_dir, 'test-0000.params'), os.path.join(module_dir, 'test-hyphens-0000.params'))
+    export_path = '{}/test-hyphens.model'.format(tmpdir)
```

Comment:
Recommend to use os.path.join.
Easy for future windows support.

Anyway, you decide.

Reply:
good point, but will fix later.